### PR TITLE
autofill-parser: flesh out generated POM file

### PR DESCRIPTION
**Pull request by msfjarvis**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## :scroll: Description
Add the standard set of fields that libraries typically ship in POM files. This results in the following diff from our currently published POM file.

```diff
diff --git current_pom.xml /home/msfjarvis/.m2/repository/com/github/androidpasswordstore/autofill-parser/1.0.0/autofill-parser-1.0.0.pom
index 032a12e1301e..5280d459e6db 100644
--- current_pom.xml
+++ /home/msfjarvis/.m2/repository/com/github/androidpasswordstore/autofill-parser/1.0.0/autofill-parser-1.0.0.pom
@@ -11,6 +11,27 @@
   <artifactId>autofill-parser</artifactId>
   <version>1.0.0</version>
   <packaging>aar</packaging>
+  <name>autofill-parser</name>
+  <description>Android library for low-level parsing of Autofill structures</description>
+  <url>https://github.com/Android-Password-Store/android-password-store</url>
+  <licenses>
+    <license>
+      <name>LGPL-3.0-only WITH LGPL-3.0-linking-exception</name>
+      <url>https://www.gnu.org/licenses/lgpl-3.0.txt</url>
+    </license>
+  </licenses>
+  <developers>
+    <developer>
+      <id>android-password-store</id>
+      <name>The Android Password Store Authors</name>
+      <email>aps@msfjarvis.dev</email>
+    </developer>
+  </developers>
+  <scm>
+    <connection>scm:git:https://github.com/Android-Password-Store/android-password-store.git</connection>
+    <developerConnection>scm:git:ssh://git@github.com:Android-Password-Store/android-password-store</developerConnection>
+    <url>https://github.com/Android-Password-Store/android-password-store</url>
+  </scm>
   <dependencies>
     <dependency>
       <groupId>org.jetbrains.kotlin</groupId>
```

## :bulb: Motivation and Context

Most of the fields were already being defined in our `gradle.properties` file but were left unused, this makes use of them.

## :green_heart: How did you test it?

Published the apsMaven publication to mavenLocal and diffed it from the POM file obtained from https://maven.msfjarvis.dev

## :pencil: Checklist

- [x] I formatted the code with the IDE's reformat action (Ctrl + Shift + L/Cmd + Shift + L)
- [x] I reviewed submitted code
- [ ] I added a [CHANGELOG](CHANGELOG.md) entry if applicable

### CODE REVIEW

**Comment by msfjarvis** [comment]
The description could use some love, I just filled it out quickly since my primary objective was to just test the POM customisation.

**Comment by fmeum** [comment]
> The description could use some love, I just filled it out quickly since my primary objective was to just test the POM customisation.

Not sure I can do much better. Here is my best bet so far:
```
Android library for AutofillService implementations that handles the low-level view structure parsing
```

File: autofill-parser/gradle.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
 GROUP=com.github.androidpasswordstore
 VERSION_NAME=1.0.0
 POM_ARTIFACT_ID=autofill-parser
+POM_ARTIFACT_DESCRIPTION=Android library for low-level parsing of Autofill structures
```
Comment (fmeum) [inline autofill-parser/gradle.properties:4]:
```suggestion
POM_ARTIFACT_DESCRIPTION=Android library for AutofillService implementations that handles the low-level view structure parsing
```

**Review by fmeum — APPROVED** [review]

File: autofill-parser/gradle.properties (hunk 1 above)
Comment (msfjarvis) [reply autofill-parser/gradle.properties:4]: Doesn't really feel like an improvement, I'll merge my version for now and we can update it when something more explanatory occurs to either of us.

**Pull request merged**